hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -57,12 +120,39 @@ def __init__(self, key: str, value: str):
         super(self.__class__, self).__init__(key, value)
 
 
+class Action:
```

Comment:
I think we can use `@dataclass` as a decorator and remove the `__init__` function